src/input/manager.rs

```diff
@@ -1205,7 +917,8 @@ impl Manager {
         device: UdevDevice,
         id: String,
     ) -> Result<(), Box<dyn Error>> {
-        log::debug!("Source device removed: {}", device.devnode());
+        let dev_name = device.name();
+        log::debug!("Source device removed: {dev_name}");
```

Comment:
If we aren't reusing dev_name then we don't need to break it out.

Reply:
> If we aren't reusing dev_name then we don't need to break it out.

Same as before: cargo fmt will transform those two lines in 4 lines where one will be );